docs/docs/guide/_sections/functions-call.md

```diff
@@ -0,0 +1,7 @@
+To call a function in an expression, rivets includes a special formatter `call`. This formatter will call the function, any formatter parameter will become an argument sent to the function. The following text binding will call the function `event.duration` with the two arguments `event;start` and `event.end`.
```

Comment:
Sorry just 2 things: typo on `event;start`, and could you add the explanation "event.duration will be called again every time `event.start` and `event.end` change"?